etc/jenkins/publish.sh

```diff
@@ -61,12 +61,12 @@ echo "======== Fetch library dependencies"
 popd
 
 echo "======== Create zip"
-ZIP_STAGING="${SCRATCH}/staging"
+ZIP_STAGING="${WORKSPACE}/staging"
 rm -rf "${ZIP_STAGING}"
 mkdir -p "${ZIP_STAGING}"
 pushd "${ZIP_STAGING}"
 cp "${ZIP_BUILD}/target/dependency/"*.jar . || exit 1
-curl "https://raw.githubusercontent.com/couchbase/build/master/license/couchbase-lite/LICENSE_${EDITION}.txt" -o LICENSE.TXT || exit 1
+cp "${WORKSPACE}/product-texts/mobile/couchbase-lite/license/LICENSE_${EDITION}.txt" .
```

Comment:
Should the file be renamed to LICENSE.TXT as before?

Reply:
I don't know.  If you would prefer that, I can certainly make the change.  I assumed the "official" name was appropriate.